Products/CMFPlone/profiles/default/catalog.xml

```diff
@@ -135,4 +135,5 @@
  <column value="portal_type"/>
  <column value="review_state"/>
  <column value="start"/>
+ <column value="MimeType"/>
```

Comment:
I know we don't really have a convention for whether to use camel case or underscores for catalog metadata columns. But the only ones that are capitalized are named after CMF accessor methods. I would probably call this `mime_type`